app/controllers/vms_controller.rb

```diff
@@ -153,10 +153,6 @@ def reset_vm
     redirect_back(fallback_location: root_path)
   end
 
-  # This controller doesn't use strong parameters
-  # https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html
-  # Because no Active Record model is being wrapped
```

Comment:
Why was this comment removed? Is it no longer relevant?

Reply:
The controller now uses strong parameters
and it also has an ActiveRecordModel now (even though its name does not quite match (vms_controller vs VirtualMachineConfig)

Reply:
Thanks for the info!